pkgs/os-specific/linux/librist/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub, meson, ninja, pkg-config }:
+
+stdenv.mkDerivation rec {
+  pname = "librist";
+  version = "0.2.6";
+
+  src = fetchFromGitHub {
+    owner = "nanake";
+    repo = "librist";
+    rev = "v${version}";
+    sha256 = "sha256-EgWEMRiBJGq7ILmRqdL7Ok6/FhZNmYgiYyWHUhTZqCA=";
+  };
+
+  mesonFlags = [
+    "--default-library=static"
+  ];
+
```

```suggestion

  buildInputs = [
    cjson
    mbedtls
  ];
```

Otherwise librist will use vendored versions:

```
Run-time dependency cjson found: NO (tried pkgconfig and cmake)
Has header "cjson/cJSON.h" : NO
Message: Using builtin cJSON library
Message: Building mbedtls
Run-time dependency mbedcrypto found: NO (tried pkgconfig and cmake)
Has header "mbedtls/aes.h" : NO
Message: Using builtin mbedtls code
```